frontend/src/openapi-schema.ts

```diff
@@ -445,6 +458,33 @@ export interface components {
       /** Error Type */
       type: string;
     };
+    /** Worker */
+    Worker: {
+      /**
+       * Id
+       * Format: uuid
+       */
+      id?: string;
+      /**
+       * Last Seen
+       * Format: date-time
+       */
+      last_seen?: string;
+      /** Name */
+      name: string;
+      /** Token */
+      token: string;
+    };
+    /** WorkerBase */
+    WorkerBase: {
```

Comment:
`id` should probably be included here

Reply:
No, only a `transcribee_backend.models.worker.Worker` has a id. The worker base doesn't

Reply:
In know, but would be good to have in the response

Reply:
Why? The worker id is only used internally in the backend. The only thing needed via the api is the token / name